src/components/Tooltip/Tooltip.jsx

```diff
@@ -138,26 +153,6 @@ Tooltip.positions = DialogPositions;
 Tooltip.animationTypes = DIALOG_ANIMATION_TYPES;
 Tooltip.themes = TOOLTIP_THEMES;
 Tooltip.justifyTypes = TOOLTIP_JUSTIFY_TYPES;
-Tooltip.defaultProps = {
```

Comment:
why?

Reply:
I think I've just change places of propTypes and defaultProps, cause they've been reversed in comparison to every other component

Reply:
I did not found it in the file in a different place, if it's exist please ignore this comment